tfutils/utils.py

```diff
@@ -173,18 +173,48 @@ def get_loss(inputs,
              outputs,
              targets,
              loss_per_case_func,
+             loss_per_case_func_params={'_outputs': 'logits', '_targets_$all': 'labels'},
              agg_func=None,
              loss_func_kwargs=None,
              agg_func_kwargs=None):
     if loss_func_kwargs is None:
         loss_func_kwargs = {}
     if not isinstance(targets, (list, tuple, np.ndarray)):
         targets = [targets]
+    targets = list(targets)
```

Comment:
OK, I take it back. I tested in incorrectly before commenting (did `[np.array(['labels', 'labels2'])]`)